src/app/page.tsx

```diff
@@ -28,7 +30,7 @@ function Home({ searchParams }: Props) {
           return;
         }
 
-        const response = await getNews(searchParams.q, selects);
+        const response = await getNews(searchParams.q, searchParams.page, selects);
```

Comment:
`searchParams.page` is unused

Reply:
change logic with page number in url, now it's used)